src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java

```diff
@@ -268,7 +268,7 @@ private static void initLogLevel(Config config) {
                     //FIXME remove the sandbox url - but I am to paranoid
                     platformInstance = mturkPlatform = new MturkPlatform(platform.user,
                             platform.password,
-                            "https://mechanicalturk.sandbox.amazonaws.com/",
+                            "https://mechanicalturk.amazonaws.com/",
```

Comment:
Hello,

i hardcoded this value that noone uses productive mturk by accident. there is platform.url which will take the value form the config file, which doesnt need a hardcoded string at all here. So platforms can still be used for productive OR sandbox.